examples/jsm/loaders/VRMLLoader.js

```diff
@@ -3275,7 +3275,7 @@ class VRMLLexer {
 
 }
 
-const CstParser = chevrotain.CstParser;
+const { CstParser } = chevrotain;
```

Comment:
Perhaps a better example of https://github.com/mrdoob/three.js/pull/26912#discussion_r1349771865. I refactored this from a static method back to a class, figuring that destructuring from `chevrotain` rather than member access would tree-shake and that seems to be the case. Why they're not equivalent for tools, I'm not sure. Perhaps it's accounting for possible getters? This case is regarded in https://esbuild.github.io/api/#tree-shaking-and-side-effects.

Reply:
In any event, I like this solution more than the previous refactoring 😇 .